### ISSUE TITLE
False positive I004 isort found an unexpected blank line in imports

### ISSUE THREAD

**Issue by pomcho555** [wemake-services/wemake-python-styleguide#2230]
### What's wrong

My lines.
```python
import base64
import io
import os
import uuid

import pandas as pd

import config
```

Reviewdog on GitHub actions say `False positive I004 isort found an unexpected blank line in imports`

However, if I run isort and flake8-isort on the same content on my local environment, I get no errors as expected.

#### Githubactions
- wemake-services/wemake-python-styleguide@0.15.3

#### Local
- flake8: 4.0.1 (flake8_isort: 4.1.1, mccabe: 0.6.1, pycodestyle: 2.8.0, pyflakes: 2.4.0) CPython 3.8.7 on
Darwin
- isort: 5.9.3

### How it should be

Need consistency against local linters. This might be caused by [flake8-isort version](https://github.com/wemake-services/wemake-python-styleguide/blob/4ab2fb81af84c8e557750dceea0f99c2c8fd4e99/pyproject.toml#L72). Should be upgrade?

#### local

```
{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.8.7",
    "system": "Darwin"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "flake8_isort",
      "version": "4.1.1"
    },
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.8.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.4.0"
    }
  ],
  "version": "4.0.1"
}
```

#### local

```
pip 21.1.3 from /Users/traveroliver/workspace/masktap-spf/venv/lib/python3.8/site-packages/pip (python 3.8)
backports.entry-points-selectable==1.1.0
distlib==0.3.3
filelock==3.3.1
flake8==4.0.1
flake8-isort==4.1.1
isort==5.9.3
mccabe==0.6.1
numpy==1.21.2
packaging==21.0
pandas==1.3.2
platformdirs==2.4.0
pluggy==1.0.0
py==1.10.0
pycodestyle==2.8.0
pyflakes==2.4.0
pyparsing==3.0.1
python-dateutil==2.8.2
pytz==2021.1
six==1.16.0
testfixtures==6.18.3
toml==0.10.2
tox==3.24.4
virtualenv==20.9.0
```

#### local
macOS Big Sur version 11.6

#### Github Actions
ubuntu-latest
